File: app/src/main/java/ru/example/newsreader/room/Entity/SourceEntity.java
package ru.example.newsreader.room.Entity;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import android.support.annotation.NonNull;

@Entity
public class SourceEntity {
    @NonNull
    @PrimaryKey
    private String name;
    private String url;

    @NonNull
    public String getName() {
        return name;
    }

    public void setName(@NonNull String name) {
        this.name = name;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: app/src/main/java/ru/example/newsreader/room/entity/SourceEntity.kt
package ru.example.newsreader.room.entity

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import android.support.annotation.NonNull

@Entity
data class SourceEntity (
        @NonNull
        @PrimaryKey
        @ColumnInfo
        var name : String,
        @ColumnInfo
        var url : String
)